testcases/kernel/syscalls/ipc/msgrcv/msgrcv08.c

```diff
@@ -47,11 +47,13 @@ const char *TCID = "msgrcv08";
 const int TST_TOTAL = 1;
 
 #if __WORDSIZE == 32
-
+#ifdef __GLIBC__
 struct msgbuf {
 	long mtype;     /* message type, must be > 0 */
 	char mtext[16]; /* message data */
 };
+#else
+#endif
 
```

Comment:
The definition in the system headers should not be exposed unless the test defines _GNU_SOURCE for both glibc and musl. Are you sure that the redefinition is not caused by local changes in your tree?